### ISSUE TITLE
ubuntu 16.04, con't find libxml2

### ISSUE THREAD

**Issue by bam241** [cyclus/cyclus#1426]
As reported in #1424 , `cmake` is not able to find `libxml2`:
```
CMake Error at CMakeLists.txt:153 (FIND_PACKAGE):
  Could not find a package configuration file provided by "LibXml2" with any
  of the following names:

    LibXml2Config.cmake
    libxml2-config.cmake

  Add the installation prefix of "LibXml2" to CMAKE_PREFIX_PATH or set
  "LibXml2_DIR" to a directory containing one of the above files.  If
  "LibXml2" provides a separate development package or SDK, be sure it has
  been installed.

```

reverting some of the change in the CMakeList.txt file allow to fix this.
```
+    if (DEPS_ROOT_DIR)
+        SET(DEPS_HINTS HINTS "${DEPS_ROOT_DIR}")
+        SET(DEPS_BIN_HINTS ${DEPS_HINTS} "${DEPS_ROOT_DIR}/bin")
+        SET(DEPS_LIB_HINTS ${DEPS_HINTS} "${DEPS_ROOT_DIR}/lib")
+        SET(DEPS_INCLUDE_HINTS HINTS "${DEPS_ROOT_DIR}/include")
+    else (DEPS_ROOT_DIR)
+        SET(DEPS_HINTS)
+        SET(DEPS_bin_HINTS)
+        SET(DEPS_LIB_HINTS)
+        SET(DEPS_INCLUDE_HINTS)
+    endif (DEPS_ROOT_DIR)

-    if (NOT DEPS_ROOT_DIR)
-        get_filename_component(compdir ${CMAKE_CXX_COMPILER} DIRECTORY)
-        get_filename_component(DEPS_ROOT_DIR ${compdir} DIRECTORY)
-    endif (NOT DEPS_ROOT_DIR)
-    SET(DEPS_HINTS HINTS "${DEPS_ROOT_DIR}")
-    SET(DEPS_BIN_HINTS ${DEPS_HINTS} "${DEPS_ROOT_DIR}/bin")
-    SET(DEPS_LIB_HINTS ${DEPS_HINTS} "${DEPS_ROOT_DIR}/lib")
-    SET(DEPS_INCLUDE_HINTS HINTS "${DEPS_ROOT_DIR}/include")
-    MESSAGE("-- Dependency Root Dir (DEPS_ROOT_DIR): ${DEPS_ROOT_DIR}")
-    MESSAGE("-- Dependency Hints (DEPS_HINTS): ${DEPS_HINTS}")
-    MESSAGE("-- Dependency Binary Hints (DEPS_BIN_HINTS): ${DEPS_BIN_HINTS}")
-    MESSAGE("-- Dependency Library Hints (DEPS_LIB_HINTS): ${DEPS_LIB_HINTS}")
-    MESSAGE("-- Dependency Include Hints (DEPS_INCLUDE_HINTS): ${DEPS_INCLUDE_HINTS}")
```
I am not sure to understand the goal of this change, so I did not PR a revert...

**Comment by bam241** [comment]
this should be fixed with #1428

**Comment by bam241** [comment]
this should be fix, closing
